src/theme.css

```
@@ -0,0 +1,106 @@
+  --success-foreground: oklch(21.03% 0.0059 138.52);
+  --surface: oklch(21.03% 0.003 151.41);
+  --surface-foreground: oklch(99.11% 0.0015 151.41);
+  --surface-secondary: oklch(25.7% 0.0023 151.41);
+  --surface-secondary-foreground: oklch(99.11% 0.0015 151.41);
+  --surface-tertiary: oklch(27.21% 0.0023 151.41);
+  --surface-tertiary-foreground: oklch(99.11% 0.0015 151.41);
+  --warning: oklch(82.03% 0.1392 64.05);
+  --warning-foreground: oklch(21.03% 0.0059 64.05);
+}
+
+@theme {
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)